### PR TITLE
docs: Add README and CODEOWNERS

### PR DESCRIPTION
**Pull request by muselesscreator**
Round one of adding a readme with contribution information and CODEOWNERS.

### CODE REVIEW

File: src/containers/LearnerDashboardHeader/hooks.test.js [hunk 1]
```diff
@@ -4,12 +4,18 @@ import { useIsCollapsed } from './hooks';
 describe('LearnerDashboardHeader hooks', () => {
   describe('useIsCollapsed', () => {
     test('large screen is not collapsed', () => {
+      const width = breakpoints.large.maxWidth + 1;
```
Comment (jansenk) [inline src/containers/LearnerDashboardHeader/hooks.test.js:7]: Should this be in this PR?

File: src/setupTest.jsx [hunk 2]
```diff
@@ -12,7 +12,7 @@ jest.mock('react', () => ({
   useRef: jest.fn((val) => ({ current: val, useRef: true })),
   useCallback: jest.fn((cb, prereqs) => ({ useCallback: { cb, prereqs } })),
   useEffect: jest.fn((cb, prereqs) => ({ useEffect: { cb, prereqs } })),
-  useMemo: jest.fn((cb, prereqs) => cb(prereqs)),
+  useMemo: jest.fn((cb, prereqs) => ({ useMemo: { cb, prereqs } })),
```
Comment (nsprenkle) [inline src/setupTest.jsx:15]: Why are code changes part of this README PR? Split out if possible

File: README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# frontend-app-learner-dashboard
+
+The Learner Dashboard app is a microfrontend (MFE) course listing experience for the OpenEdx Learning Management System.  This experience was designed to provide a clean and functional interface to allow learners to view all of their open enrollments, as well as take relevant actions on those enrolments.  It also serves as host to a number of exposed "widget" containers to provide upsell and discovery widgets as sidebar/footer components.
```
Comment (nsprenkle) [inline README.md:3]: We should standardize around the name "Learner Home", particularly because we plan on renaming this repo.

**Review by nsprenkle — COMMENTED** [review]
Note to remove code changes from a documentation PR. Also should add CODEOWNERS file

**Comment by muselesscreator** [comment]
Note: do not merge before release is finished (11/18)

**Review by nsprenkle — APPROVED** [review]

**Pull request merged**